src/ngx_http_lua_util.c

```diff
@@ -2008,8 +2008,45 @@ ngx_http_lua_escape_uri(u_char *dst, u_char *src, size_t size, ngx_uint_t type)
 
                     /* mail_auth is the same as memcached */
 
```

Comment:
We should summary here in a comment about exactly what characters are to be escaped, like `/* " ", "%", %00-%1F */` for the memcached case above.